catalog/admin/module/catalog-admin-module-sources/src/main/webapp/templates/checkbox.hbs

```diff
@@ -0,0 +1 @@
+<span class="fa fa-square-o"></span>
```

Comment:
For future reference, probably don't need a separate handlebars file for a simple span